integration/crds/lightweight/DanmNet.yaml

```diff
@@ -1,116 +1,126 @@
+            type: string
+          metadata:
+            type: object
+          spec:
+            properties:
+              AllowedTenants:
+                items:
                   type: string
-                vxlan:
-                  type: integer
-                  format: int32
-                  minimum: 1
-                  maximum: 16777214
-                vlan:
-                  type: integer
-                  format: int32
-                  minimum: 1
-                  maximum: 4094
-                rt_tables:
-                  type: integer
-                  format: int32
-                  minimum: 0
-                  maximum: 255
-                net6:
-                  oneOf:
-                  - type: string
-                    maxLength: 0
-                  - type: string
-                    format: cidr
-                    pattern: ':'
-                allocation_pool_v6:
-                  properties:
-                    start:
-                      oneOf:
-                      - type: string
-                        maxLength: 0
-                      - type: string
-                        format: ipv6
-                    end:
-                      oneOf:
-                      - type: string
-                        maxLength: 0
-                      - type: string
-                        format: ipv6
-                    lastIp:
-                      oneOf:
-                      - type: string
-                        maxLength: 0
-                      - type: string
-                        format: cidr
-                        pattern: ':'
-                    cidr:
-                      oneOf:
-                      - type: string
-                        maxLength: 0
-                      - type: string
-                        format: cidr
-                        pattern: ':'
-                routes:
-                  type: object
-                routes6:
-                  type: object
+                type: array
+              NetworkID:
+                type: string
+              NetworkType:
+                type: string
+              Options:
+                properties:
+                  alloc:
+                    description: bit array of tracking address allocation
+                    type: string
+                  alloc6:
+                    description: bit array tracking IPv6 allocations
+                    type: string
+                  allocation_pool:
+                    description: subset of the IPv4 subnet from which IPs can be allocated
+                    properties:
+                      end:
+                        type: string
+                      lastIp:
+                        type: string
+                      start:
+                        type: string
+                    required:
```

Comment:
dont add backward incompatble API changes not present on the original IP: allocation pools can be unbound in either direction

Reply:
you can explain this ???

Reply:
you designated parameters as mandatory which weren't in the original API spec, therefore you made a backward incompatible change during the upgrade